src/redux/sagas/expenditures/releaseExpenditureStage.ts

```diff
@@ -111,6 +114,19 @@ function* releaseExpenditureStage({
         txHash,
       });
     }
+    const slotToClaim = expenditure.slots.find((slot) => slot.id === slotId);
+    // Should never be undefined, but still
+    if (slotToClaim) {
+      const claimableSlots = getImmediatelyClaimableSlots([slotToClaim]);
+
+      yield claimExpenditurePayouts({
+        colonyAddress,
+        claimablePayouts: getPayoutsWithSlotIdsFromSlots(claimableSlots),
```

Comment:
Judging by the fact you've got to call this every time you call the helper saga, you could potentially move it to the saga itself.